server/methods.js

```diff
@@ -89,15 +89,8 @@ Meteor.startup(function() {
     },
     deleteNotification: function(notification){
       if(Meteor.user()){
```

Comment:
We should be checking that notification is what we expect. Meaning you should do something like:

```
check({
  _id: String,
  ownerId: String,
  createdAt: Date,
  // rest of the properties in a notification
});
```